diff --git a/CMakeLists.txt b/CMakeLists.txt
index f580d20..500e832 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,7 +24,7 @@ find_package(PNG REQUIRED)
 find_package(JPEG REQUIRED)
 find_package(TIFF REQUIRED)
 
-add_subdirectory(elibs)
+#add_subdirectory(elibs)
 
 include_directories(SYSTEM ${CMAKE_SOURCE_DIR}/elibs/mve/libs/)
 link_directories(
diff --git a/apps/raycast/CMakeLists.txt b/apps/raycast/CMakeLists.txt
index 767a3da..367339e 100644
--- a/apps/raycast/CMakeLists.txt
+++ b/apps/raycast/CMakeLists.txt
@@ -4,5 +4,5 @@ file(GLOB SOURCES "*.cpp")
 set(BIN raycast)
 
 add_executable(${BIN} ${SOURCES})
-add_dependencies(${BIN} ext_mve)
+#add_dependencies(${BIN} ext_mve)
 target_link_libraries(${BIN} -lmve -lmve_util ${JPEG_LIBRARIES} ${PNG_LIBRARIES} ${TIFF_LIBRARIES})
